my-app/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react'
+import gameLogic from './Logic.js';
+
+class Gamepiece {
+  constructor(x,y) {
+    this.x = x
+    this.y = y
+    this.value = 'blank'
+  }
+}
+
+class GameBoard {
+  constructor( size ){
+    this.size = size
+    this.board = []
+    this.initializeBoard()
+  }
+
+  initializeBoard(){
+    for(let x = 0; x < this.size; x++){
+      this.board[x] = []
+      for (let y = 0; y < this.size; y++){
+        this.board[x][y] = new Gamepiece(x,y)
+      }
+    }
+    return this.board
+  }
+}
+
+class Board extends Component {
+  constructor() {
+    super();
+    this.state = {
+      blackScore: 0,
+      whiteScore: 0,
+      whiteIsNext: false,
+      gameBoard: new GameBoard(19),
+    };
+  }
+
+  handleClick(cell){
+    this.setState(gameLogic(this.state, cell))
+  }
+
+  switchTurn(state){
+    let newState = this.state
+    newState.whiteIsNext ? newState.whiteIsNext = false : newState.whiteIsNext = true;
+    this.setState(newState)
+
+  }
+
+  render() {
+    let rows = this.state.gameBoard.board.map( (row,index) => {
+      let cells = [];
+      for (let i in row) {
```

Comment:
This for-in loop raises a linting error because if that object were inheriting any keys from the prototype it would loop through those too without filtering them out. You can make sure each key belongs to the object itself using hasOwnProperty.

for(let i in row) {
   &nbsp;&nbsp;&nbsp;if(row.hasOwnProperty(i)){
     &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;//etc...
&nbsp;&nbsp;&nbsp;}
}